Add 0.16.0 release notes to devel

dfarrell07 opened the pull request.

tpantelis committed:
Add release section for 0.16.0

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Document 'subctl cloud prepare azure' command `air-gapped` flag

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

skitt committed:
Add release notes for subctl#797

subctl show versions now shows the versions of the metrics proxy and
plugin syncer components.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

skitt committed:
Add release notes for subctl#723

subctl is now built for ARM Macs (Darwin arm64).

Signed-off-by: Stephen Kitt <skitt@redhat.com>

sridhargaddam committed:
Add release notes for couple of PRs

Includes the release notes for the following fixes.

* Submariner now uses case-insensitive comparison while parsing
  CNI names.
* subctl gather now collects Metrics proxy pod logs in a Globalnet
  deployment.
* Submariner Gateway pod now skips invoking cableEngine cleanup during
  termination, as this is handled by the Route agent during gateway migration.
* Fixed issue which caused the IPsec pluto process to crash when the remote
  endpoint was unstable.
* Submariner now handles out-of-order remote endpoint notifications properly
  in Globalnet component.

Related to: https://github.com/submariner-io/submariner/pull/2486
Related to: https://github.com/submariner-io/subctl/pull/770
Related to: https://github.com/submariner-io/submariner/pull/2499
Related to: https://github.com/submariner-io/submariner/pull/2517
Related to: https://github.com/submariner-io/submariner/pull/2532
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

aswinsuryan committed:
Add release note for submariner-io/submariner#2515

Signed-off-by: Aswin Suryanarayanan <aswinsuryan@gmail.com>

yboaron committed:
Add release notes for submariner#2543

Reduced data path downtime with Libreswan cable driver when gateway
pod restarts.

Signed-off-by: Yossi Boaron <yboaron@redhat.com>

tpantelis committed:
Add release note for EndpointSlice changes

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Release note for Globalnet status condition change

by https://github.com/submariner-io/submariner/pull/2487.

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

sridhargaddam committed:
Add release notes for Submariner PR#2651

Related to: https://github.com/submariner-io/submariner/pull/2651
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

skitt committed:
Add release notes and documentation for subctl upgrade

Signed-off-by: Stephen Kitt <skitt@redhat.com>

sridhargaddam committed:
Add release notes for subctl#886

Release notes for the following PR.
https://github.com/submariner-io/subctl/pull/886

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

sridhargaddam committed:
Add release notes for submariner-operator PR#2769

Release notes for PR
https://github.com/submariner-io/submariner-operator/pull/2769
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

sridhargaddam committed:
Add release notes for Submariner PR#2599

Release notes for https://github.com/submariner-io/submariner/pull/2599

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

tpantelis committed:
Add release note for lighthouse PR#1380

Release note for https://github.com/submariner-io/lighthouse/pull/1308.

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

sridhargaddam committed:
Release notes for flannel cni PR#2785

https://github.com/submariner-io/submariner-operator/pull/2785

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

tpantelis committed:
Add release notes for lighthouse PR#1241

Release notes for https://github.com/submariner-io/lighthouse/pull/1241

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Remove network plugin syncer reference in RN

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Add release notes for lighthouse PR#1344

Release notes for https://github.com/submariner-io/lighthouse/pull/1344

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Add release notes for subctl PR#868

Release notes for https://github.com/submariner-io/subctl/pull/868

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Add release notes for submariner PR#2637

Release notes for https://github.com/submariner-io/submariner/pull/2637

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Organize 0.16 release notes

...into "New features" and "Other changes" sections as was done
for past releases.

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

sridhargaddam committed:
Release notes for IPsec connection issues

This PR includes release notes for the following fixes
https://github.com/submariner-io/submariner/pull/2708
https://github.com/submariner-io/submariner/pull/2710

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

sridhargaddam committed:
Release notes for podCIDR detection

This PR includes release notes for
https://github.com/submariner-io/submariner-operator/pull/2770

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

tpantelis committed:
Add release note for lighthouse PR#1387

Release note for https://github.com/submariner-io/lighthouse/pull/1387

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

skitt committed:
Describe the upgrade issue involving OVN sockets

Signed-off-by: Stephen Kitt <skitt@redhat.com>

tpantelis committed:
Add release note for lighthouse PR#818

Release note for https://github.com/submariner-io/lighthouse/pull/818

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Add release note for submariner PR#2576

Release note for https://github.com/submariner-io/submariner/pull/2576

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

tpantelis committed:
Add release note for submariner PR#2526

Release note for https://github.com/submariner-io/submariner/pull/2526

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

dfarrell07 commented:
> This branch cannot be rebased due to conflicts

???

There are no commit on devel and not on this branch.

> git log origin/devel ^merge_relnotes16

I guess we have some config that requires rebasing, not merging.

dfarrell07 committed:
Add release date to 0.16.0 release notes

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

maayanf24 commented on src/content/community/releases/_index.en.md:
```diff
@@ -461,6 +521,8 @@ via the [Submariner addon](https://github.com/open-cluster-management/submariner
   port name and protocol. For a ClusterIP Service, this resolves to the port number and the domain name. For a Headless Service,
   the name resolves to multiple answers, one for each Pod backing the Service.
+* The CNI detection method in Submariner Operator is now improved to detect the Flannel CNI, even when the Flannel configMap
+  is missing from the cluster.
```
This is not related to 0.16.0 release notes, or did it got to the wrong place?

dfarrell07 replied:
@sridhargaddam sent this relnote to the 0.16 branch here: https://github.com/submariner-io/submariner-website/pull/1020

tpantelis replied:
Apparently it was inadvertently put in the v0.10.1 section.

sridhargaddam replied:
Oops, thanks for spotting it @maayanf24 and @dfarrell07.

Shall I send an updated PR to fix it in the release-notes-0.16 branch first?

sridhargaddam replied:
Pushed a fix on the release-0.16 branch - https://github.com/submariner-io/submariner-website/pull/1059

sridhargaddam committed:
Fix release notes for Flannel CNI

The fix related to Flannel CNI was inadvertently added to
a wrong section. This PR fixes it.

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>
